src/Toolchain/ToolchainProvider.ts

```diff
@@ -243,9 +243,10 @@ export class ToolchainProvider implements vscode.TreeDataProvider<BaseNode> {
     gToolchainEnvMap[backendName]
         .uninstall(tnode.toolchain)
         .then(() => notifyUninstalled(), () => notifyError());
```

Comment:
Shouldn't it be 'return false' for notifyError() case..?

Reply:
Q. Do you define the function to return 'true' if it has tried running, no matter what the result is?

Reply:
The return value of this function does not mean much. Only valid case is when testing, but the test for installation and uninstall does not reach the case you mentioned.

Reply:
> The return value of this function does not mean much.

If so, how about returning void? Or if it's gonna return boolean, could you elaborate it somewhere (comment of so)

Reply:
Sure. I will fix it in next PR. :)